libmscore/xmlreader.cpp

```diff
@@ -185,7 +185,6 @@ Fraction XmlReader::readFraction()
       if (!s.isEmpty()) {
             int i = s.indexOf('/');
             if (i == -1) {
-                  qDebug("reading ticks <%s>", qPrintable(s));
```

Comment:
It is seen in Debug builds only though, isn't it?

Reply:
[“This function does nothing if `QT_NO_DEBUG_OUTPUT` was defined during compilation.”](https://doc.qt.io/Qt-5/qtglobal.html#qDebug)

I see. `QT_NO_DEBUG_OUTPUT` is defined for RELEASE builds. For some reason, Debian builds are neither DEBUG nor RELEASE but "" (empty string).

Feel free to disregard this or merge it, I don’t mind either way.

Reply:
I guess Debian should do "RELEASE" builds then?